ee/fe/modules/340-monitoring-applications/templates/base-service-monitor.yaml

```diff
@@ -22,27 +22,27 @@
   replacement: $1
   targetLabel: endpoint
 
-# HTTP Scheme
+  # HTTP Scheme
   {{ if eq $schema "http" }}
 - sourceLabels: [__meta_kubernetes_endpoint_port_name]
   regex: "https-metrics"
   action: drop
 - sourceLabels:
-  - __meta_kubernetes_service_annotationpresent_prometheus_deckhouse_io_port
-  - __meta_kubernetes_service_annotationpresent_prometheus_deckhouse_io_tls
-  - __meta_kubernetes_endpoint_port_name
+    - __meta_kubernetes_service_annotationpresent_prometheus_deckhouse_io_port
+    - __meta_kubernetes_service_annotationpresent_prometheus_deckhouse_io_tls
+    - __meta_kubernetes_endpoint_port_name
   regex: "^true;;(.*)|;;http-metrics$"
   action: keep
 
-# HTTPS Scheme
+  # HTTPS Scheme
```

Comment:
Why we move this one step right ? I think it is Goland's work.

Reply:
This one I moved this one intentionally to be in line with the condition.